cli/src/pcluster/validators/scheduler_plugin_validators.py

```diff
@@ -8,9 +8,16 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
+import operator
+import re
+from collections import namedtuple
+
+from pkg_resources import packaging
 
 from pcluster.validators.common import FailureLevel, Validator
 
+CliRequirement = namedtuple("Requirement", "operator version")
```

Comment:
we could move this to SupportedVersionsValidator class

Reply:
Moved inside validation function